spec/iot-js-spec.md

```diff
@@ -160,7 +153,6 @@ interface OicRequestEvent : Event {
   readonly attribute USVString source;  // device uuid of the OIC client making the request
   readonly attribute USVString target;  // id of the target resource
   readonly attribute OicResourceRepresentation? res;  // for “create” and “update” only
```

Comment:
You need to change the comment to say ```// for "update" only``` and you also need to add
```
readonly attribute OicResourceInit? init; // for "create" only
```
because in the case of create we need the metadata as well (url, the types, the interfaces, and the discoverable/observable flags).